ortb/request_validator.go

```diff
@@ -88,22 +88,27 @@ func (srv *standardRequestValidator) validateImpExt(imp *openrtb_ext.ImpWrapper,
 	prebid := impExt.GetOrCreatePrebid()
 	prebidModified := false
 
+	bidderPromote := false
+
 	if prebid.Bidder == nil {
 		prebid.Bidder = make(map[string]json.RawMessage)
+		bidderPromote = true
 	}
 
 	ext := impExt.GetExt()
 	extModified := false
 
 	// promote imp[].ext.BIDDER to newer imp[].ext.prebid.bidder.BIDDER location, with the later taking precedence
-	for k, v := range ext {
-		if openrtb_ext.IsPotentialBidder(k) {
-			if _, exists := prebid.Bidder[k]; !exists {
-				prebid.Bidder[k] = v
-				prebidModified = true
+	if bidderPromote {
+		for k, v := range ext {
+			if openrtb_ext.IsPotentialBidder(k) {
+				if _, exists := prebid.Bidder[k]; !exists {
+					prebid.Bidder[k] = v
+					prebidModified = true
+				}
+				delete(ext, k)
+				extModified = true
 			}
-			delete(ext, k)
-			extModified = true
 		}
 	}
```

Comment:
One more option would be to allow for an empty `imp.ext.prebid.bidder` object i.e. `{}` - and promote in that case too, but Perhaps helps in case someone was unintentionally adding this object into requests, but not filling it, but I don't think there are clients that do this...

Reply:
> This is what is called out in the issue and what PBS-Java does. Just confirming that we are ok with that approach now.

Yes. This is the design of the feature and the next phase of deprecating `imp[].ext` for bidder parameters. It should've never been designed this way, but here we are.